.github/scripts/release/create_changelog.sh

```diff
@@ -5,49 +5,32 @@ set -o errexit
 set -E
 set -o pipefail
 
-RELEASE_VERSION=$1
-PREVIOUS_RELEASE=$2
+CURRENT_RELEASE_TAG=$1
+DOCKER_IMAGE_URL=$2
+LAST_RELEASE_TAG=$3
 
-
-if [ "${PREVIOUS_RELEASE}"  == "" ]
+if [ "${LAST_RELEASE_TAG}"  == "" ]
 then
-  PREVIOUS_RELEASE=$(git describe --tags --abbrev=0)
+  LAST_RELEASE_TAG=$(git describe --tags --abbrev=0)
 fi
 
-REPOSITORY=${REPOSITORY:-kyma-project/template-operator}
-GITHUB_URL=https://api.github.com/repos/${REPOSITORY}
-GITHUB_AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
+GITHUB_URL=https://api.github.com/repos/$CODE_REPOSITORY
+GITHUB_AUTH_HEADER="Authorization: Bearer $GITHUB_TOKEN"
 CHANGELOG_FILE="CHANGELOG.md"
 
-echo "## What has changed" >> ${CHANGELOG_FILE}
-git log "${PREVIOUS_RELEASE}"..HEAD --pretty=tformat:"%h" --reverse | while read -r commit
+git log "$LAST_RELEASE_TAG"..HEAD --pretty=tformat:"%h" --reverse | while read -r commit
 do
-    COMMIT_AUTHOR=$(curl -H "${GITHUB_AUTH_HEADER}" -sS "${GITHUB_URL}/commits/${commit}" | jq -r '.author.login')
+    COMMIT_AUTHOR=$(curl -H "$GITHUB_AUTH_HEADER" -sS "$GITHUB_URL"/commits/"$commit" | jq -r '.author.login')
```

Comment:
Was it necessary to add the quotation marks (`"`)? In line 30 below we have a similar usage (large string with multiple vars) where we don't put the single vars in quotes.